### ISSUE TITLE
Conflict with generator-karma version. Need update to generator-karma@>=0.8.2

### ISSUE THREAD

**Issue by wa6smn** [iamblue/generator-angular-ui-router#17]
npm ERR! peerinvalid The package generator-karma does not satisfy its siblings' peerDependencies require                    ments!
npm ERR! peerinvalid Peer generator-angular@0.10.0 wants generator-karma@>=0.8.2
npm ERR! peerinvalid Peer generator-angular-ui-router@0.5.3 wants generator-karma@~0.5.0

npm ERR! System Linux 3.17.7-200.fc20.x86_64
npm ERR! command "node" "/bin/npm" "install" "-g" "generator-angular-ui-router"
npm ERR! cwd /root
npm ERR! node -v v0.10.33
npm ERR! npm -v 1.3.6
npm ERR! code EPEERINVALID

**Comment by lfernandez55** [comment]
I'm getting this same error

**Comment by KidA001** [comment]
@lfernandez55 @wa6smn - You need to downgrade your Karma version for this to work.

@iamblue - Are there any plans to update ui-router generator to work with the newer versions of karma?

**Comment by bingomanatee** [comment]
npm ERR! peerinvalid The package generator-karma does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer generator-angular@0.11.1 wants generator-karma@>=0.9.0
npm ERR! peerinvalid Peer generator-angular-ui-router@0.5.3 wants generator-karma@~0.5.0

yes please make this work with curent stack

**Comment by shavo007** [comment]
i had to uninstall angular generator first!
 npm uninstall -g  generator-angular

**Comment by corehook** [comment]
@shavo007 thx! helped to me!

**Comment by calebhskim** [comment]
UNMET PEER DEPENDENCY generator-karma@~0.5.0
└── UNMET PEER DEPENDENCY yo@>=1.0.0-rc.1.1

npm WARN EPEERINVALID generator-angular-ui-router@0.5.3 requires a peer of generator-karma@~0.5.0 but none was installed.
npm WARN EPEERINVALID generator-angular-ui-router@0.5.3 requires a peer of yo@>=1.0.0-rc.1.1 but none was installed.

An update would be very much appreciated. Thanks!

**Comment by wahengchang** [comment]
need help ~

**Comment by emilycoco** [comment]
npm uninstall -g generator-karma
npm uninstall -g generator-angular
npm install -g generator-angular-ui-router

**Comment by victorsosa** [comment]
Guys, I know the patch for this was submitted a long time ago , but this issue still happening; Do we need a new release maybe???

==============
npm WARN generator-angular-ui-router@0.5.3 requires a peer of generator-karma@~0.5.0 but none was installed.

**Comment by victorsosa** [comment]
Yep confirmed; we need a new release with this fix; 0.5.3 doesn't have the patch

https://github.com/iamblue/generator-angular-ui-router/releases

**Comment by sahilkul95** [comment]
npm WARN angular-ui-router@1.0.3 requires a peer of angular@>=1.2.0 but none was installed.

I got many of such errors for karma@..... when running npm install grunt --save
![npm grunt error](https://user-images.githubusercontent.com/28224192/30743967-8ea12e2e-9fbd-11e7-88ab-f80d5d6bb286.png)

Please help me resolve it

**Comment by wa6smn** [comment]
I abandoned the software. Not suitable for use in my customer's product.

Sent from MailDroid

-----Original Message-----
